frontend/src/app/header/header.component.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { Component } from '@angular/core';
+import { TestBed, inject, async } from '@angular/core/testing';
+import { HttpModule } from '@angular/http';
+import { NgbModule } from '@ng-bootstrap/ng-bootstrap';
+
+import { HeaderComponent } from './header.component';
+import { AuthService } from './auth.service';
```

Comment:
from Travis:
```
$ ng test --single-run --code-coverage --browsers HeadlessChrome
Your global Angular CLI version (1.4.4) is greater than your local
version (1.0.0). The local Angular CLI version is used.
To disable this warning use "ng set --global warnings.versionMismatch=false".
02 10 2017 17:09:49.581:ERROR [karma]: Error: Module not found: Error: Can't resolve './auth.service' in '/home/travis/build/CodeForPoznan/volontulo/frontend/src/app/header'
    at Compiler.compiler.plugin (/home/travis/build/CodeForPoznan/volontulo/frontend/node_modules/@angular/cli/plugins/karma-webpack-throw-error.js:10:23)
    at Compiler.applyPlugins (/home/travis/build/CodeForPoznan/volontulo/frontend/node_modules/tapable/lib/Tapable.js:61:14)
    at Watching._done (/home/travis/build/CodeForPoznan/volontulo/frontend/node_modules/webpack/lib/Compiler.js:91:17)
    at onCompiled (/home/travis/build/CodeForPoznan/volontulo/frontend/node_modules/webpack/lib/Compiler.js:50:17)
    at /home/travis/build/CodeForPoznan/volontulo/frontend/node_modules/webpack/lib/Compiler.js:480:13
    at next (/home/travis/build/CodeForPoznan/volontulo/frontend/node_modules/tapable/lib/Tapable.js:202:11)
    at Compiler.<anonymous> (/home/travis/build/CodeForPoznan/volontulo/frontend/node_modules/webpack/lib/CachePlugin.js:62:5)
    at Compiler.applyPluginsAsyncSeries (/home/travis/build/CodeForPoznan/volontulo/frontend/node_modules/tapable/lib/Tapable.js:206:13)
    at /home/travis/build/CodeForPoznan/volontulo/frontend/node_modules/webpack/lib/Compiler.js:477:10
    at Compilation.applyPluginsAsyncSeries (/home/travis/build/CodeForPoznan/volontulo/frontend/node_modules/tapable/lib/Tapable.js:195:46)
    at self.applyPluginsAsync.err (/home/travis/build/CodeForPoznan/volontulo/frontend/node_modules/webpack/lib/Compilation.js:640:19)
    at next (/home/travis/build/CodeForPoznan/volontulo/frontend/node_modules/tapable/lib/Tapable.js:202:11)
    at Compilation.<anonymous> (/home/travis/build/CodeForPoznan/volontulo/frontend/node_modules/webpack/lib/ProgressPlugin.js:122:5)
    at Compilation.applyPluginsAsyncSeries (/home/travis/build/CodeForPoznan/volontulo/frontend/node_modules/tapable/lib/Tapable.js:206:13)
    at self.applyPluginsAsync.err (/home/travis/build/CodeForPoznan/volontulo/frontend/node_modules/webpack/lib/Compilation.js:631:11)
    at next (/home/travis/build/CodeForPoznan/volontulo/frontend/node_modules/tapable/lib/Tapable.js:202:11)
The command "ng test --single-run --code-coverage --browsers HeadlessChrome" exited with 1.
```

this file is in `frontend/src/app/header` directory and `auth.service` is in `frontend/src/app` so import should looks like:
```
import { AuthService } from '../auth.service';
```